src/wrapper/clap-juce-wrapper.cpp

```diff
@@ -1559,40 +1583,84 @@ class ClapJuceWrapper : public clap::helpers::Plugin<
             {
                 for (uint32_t ch = 0; ch < process->audio_inputs[idx].channel_count; ++ch)
                 {
-                    auto *ic = process->audio_inputs[idx].data32[ch] + n;
-                    if (inputChannels < outputChannels)
+                    if (supportsDouble && process->audio_outputs[idx].data64 != nullptr)
```

Comment:
I don't think it makes a difference logically, but it seems a little bit odd from the "reader's" perspective to be checking the `process->audio_outputs` pointers, when the body of this if statement is mostly concerned with `process->audio_inputs`.

Reply:
Oh no that’s a mistake for sure! I’ll fix and merge thanks!